customTests/azure_common.nhc

```diff
@@ -45,7 +46,15 @@ function background_cleanup() {
 
 
 function check_all_reduce_dependencies() {
-   if [[ ! -f $ALL_REDUCE_PATH ]]; then
+
+   # Default to Nvidia NCCL-Test
+   REDUCE_PATH=$ALL_REDUCE_PATH
+   if lspci | grep -iq AMD ; then
+      # Check if AMD RCCL-Test is installed
+      REDUCE_PATH=$RCCL_ALL_REDUCE_PATH 
+   fi
+
+   if [[ ! -f $REDUCE_PATH ]]; then
       die 1 "$FUNCNAME: all reduce executable path: $ALL_REDUCE_PATH not found. Ensure nccl-tests is installed and built in. FaultCode: NHCNA"
```

Comment:
Should the die message use `REDUCE_PATH`?

Reply:
Addressed